vdsm.spec.in

```diff
@@ -375,10 +375,12 @@ Requires:       ethtool
 Requires:       iproute
 Requires:       iproute-tc
 Requires:       ovirt-openvswitch >= 2.15
+%if 0%{?rhel} < 9
 # Workaround for BZ#1966143
 Requires:       ovirt-python-openvswitch >= 2.15
-Requires:       nmstate >= 1.0
 Requires:       nmstate-plugin-ovsdb
```

Comment:
Agreed, the `if` should really be only around `nmstate-plugin-ovsdb`

Reply:
IIUC BZ#1966143 says `vdsm.spec.in` started including `Requires ovirt-openvswitch >= 2.15` because of adding `Requires: nmstate-plugin-ovsdb` which caused unwanted openvswitch versions to be installed by the rpm build. But since rhel9, the rpm build will no longer be adding `nmstate-plugin-ovsdb` so unwanted `openvswitch` would not be installed.

Reply:
We can give it a shot, it seems like the nmstate-2.1 no longer recommends `python3dist(ovs)` however we still need to be careful around that.

Reply:
Do you think if ost with rhel8 and rhel9 passes with this patch we are in the clear?

Reply:
Requiring specific version of ovirt-python-openvswitch looks like a workaround for BZ#1966143 but it is not clear from the spec. A comment can refer to one line or
a block. If the workaround was requiring >= 2.15, and now we don't need a workaround,
the fix should be removing the special version, but this change remove the entire
package.

Reply:
> Do you think if ost with rhel and cs on 8 and 9 passes with this patch we are in the clear?

El8 is fine, there we should just check something like `rpm -q --requires vdsm-network` to see if the dependency is gone. 
For el9s we should try to install `vdsm-network` with this change and if we require the `ovirt-openvswitch` there will be conflicts.

Reply:
> Requiring specific version of ovirt-python-openvswitch looks like a workaround for BZ#1966143 but it is not clear from the spec. A comment can refer to one line or a block. If the workaround was requiring >= 2.15, and now we don't need a workaround, the fix should be removing the special version, but this change remove the entire package.

Done